File: src/main/java/com/example/reactcypressexperiment/Book.java
package com.example.reactcypressexperiment;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter @Setter @NoArgsConstructor @AllArgsConstructor @Builder
public class Book {
    private int id;
    private String author;
    private String title;
}

File: src/main/java/com/example/reactcypressexperiment/Book.kt
package com.example.reactcypressexperiment

data class Book(
    var id: Int = NO_ID,
    val author: String? = null,
    val title: String? = null,
    val recommendedReadingAge: Int? = null

) {
    companion object {
        const val NO_ID = -1
    }
}
